bcgov/HOUS-permit-portal
# Handle invalid invitation token

## Description

Refactors invitation flow so the accept invitation screen fetches the invited user by the token. This allows us to detect if the invitation token is valid and show the user an error if not.

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Implements [BPHH-1348](https://hous-bssb.atlassian.net/browse/BPHH-1348)

## Steps to QA

1. Invite a reviewer or review manager. Open the invitation.
2. Verify the accept invitation screen shows the correct information.
3. Change the token in the URL to something random. Verify the user sees an error message.

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
